### PR TITLE
chore(Dockerfile): cleanup after installation a bit more than before and keep copyright / license files

### PR DESCRIPTION
**Pull request by helgi**
Brings the Dockerfile in line with Controller, makes it easier to move to Python 3

### CODE REVIEW

File: rootfs/Dockerfile [hunk 1]
```diff
@@ -1,27 +1,35 @@
...
-		gcc \
-		libffi-dev \
-		libssl-dev \
-		musl-dev \
-		python-dev \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc
+RUN buildDeps='gcc git libffi-dev libssl-dev musl-dev python-dev python-pip python-wheel python-setuptools'; \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
```
Comment (mboersma) [inline rootfs/Dockerfile:5]: Could you add `DEBIAN_FRONTEND=noninteractive` here?

File: rootfs/Dockerfile [hunk 2]
```diff
@@ -1,27 +1,35 @@
...
+        $buildDeps \
+        libffi6 \
+        libssl1.0.0 \
+        musl \
```
Comment (mboersma) [inline rootfs/Dockerfile:9]: Does this actually use `musl`? I thought that would only be needed on alpine, and deis/base is all about ubuntu-slim now as you know.

**Review by mboersma — CHANGES_REQUESTED** [review]

File: rootfs/Dockerfile (hunk 2 above)
Comment (helgi) [reply rootfs/Dockerfile:9]: Was there before so I didn't really want to change dependencies randomly

**Comment by mboersma** [comment]
When I build this locally I see this error:

```
Step 2 : RUN buildDeps='gcc git libffi-dev libssl-dev musl-dev python-dev python-pip python-wheel python-setuptools';     apt-get update &&     apt-get install -y --no-install-recommends         $buildDeps         libffi6         libssl1.0.0         musl         python   pip install --disable-pip-version-check --no-cache-dir docker-py==1.10.3 &&     apt-get purge -y --auto-remove $buildDeps &&     apt-get autoremove -y &&     apt-get clean -y &&     COPYRIGHT_TAR='/usr/share/copyrights.tar';     gunzip $COPYRIGHT_TAR.gz; tar -rf $COPYRIGHT_TAR /usr/share/doc/*/copyright; gzip $COPYRIGHT_TAR &&     rm -rf         /usr/share/doc         /usr/share/man         /usr/share/info         /usr/share/locale         /var/lib/apt/lists/*         /var/log/*         /var/cache/debconf/*         /etc/systemd         /lib/lsb         /lib/udev         /usr/lib/x86_64-linux-gnu/gconv/IBM*         /usr/lib/x86_64-linux-gnu/gconv/EBC* &&     bash -c "mkdir -p /usr/share/man/man{1..8}"
 ---> Running in db4f699c64d2
Get:1 http://archive.ubuntu.com/ubuntu xenial InRelease [247 kB]
Get:2 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [95.7 kB]
Get:3 http://archive.ubuntu.com/ubuntu xenial-security InRelease [94.5 kB]
Get:4 http://archive.ubuntu.com/ubuntu xenial/main amd64 Packages [1558 kB]
Get:5 http://archive.ubuntu.com/ubuntu xenial/restricted amd64 Packages [14.1 kB]
Get:6 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages [9827 kB]
Get:7 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages [497 kB]
Get:8 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages [411 kB]
Get:9 http://archive.ubuntu.com/ubuntu xenial-security/main amd64 Packages [177 kB]
Get:10 http://archive.ubuntu.com/ubuntu xenial-security/universe amd64 Packages [49.5 kB]
Fetched 13.0 MB in 13s (968 kB/s)
Reading package lists...
E: Command line option --disable-pip-version-check is not understood in combination with the other options
gzip: .gz.gz: No such file or directory
tar: /usr/share/doc/*/copyright: Cannot read: Bad file descriptor
tar: At beginning of tape, quitting now
tar: Error is not recoverable: exiting now
��W ---> 4841430b1a58
```

Maybe the packaged `pip` version is too old to know about `--disable-pip-version-check`?

**Comment by helgi** [comment]
@mboersma I fixed the main issue so now you can build - missing `&&`

File: rootfs/Dockerfile (hunk 1 above)
Comment (helgi) [reply rootfs/Dockerfile:5]: Like with Controller, lets do that later when we have the actual log lines

**Review by mboersma — APPROVED** [review]

**Pull request merged**